scripts/setup-centos9.sh

```diff
@@ -78,7 +78,7 @@ function install_velox_deps_from_dnf {
   dnf_install libevent-devel \
     openssl-devel re2-devel libzstd-devel lz4-devel double-conversion-devel \
     libdwarf-devel elfutils-libelf-devel curl-devel libicu-devel bison flex \
-    libsodium-devel zlib-devel
+    libsodium-devel zlib-devel geos-devel
```

Comment:
Can the geos versions be pinned ?

Reply:
The current version is 3.10.1-1.el9 (an old version). I can replace it with `geos-devel-3.10.1-1.el9`. Does that work for you? Also, do you want to pin the version for the Ubuntu setup?

Reply:
Pinning to some top level like 3.10.1 is good enough :)